pages/docs/gentype/latest/getting-started.mdx

```diff
@@ -50,7 +41,7 @@ structure:
 ```js
   //...
   "gentypeconfig": {
-    "language": "typescript" | "flow" | "untyped",
+    "language": "typescript",
```

Comment:
Not for now, but should we remove this configuration option all together now that it just supports one? Or am I misunderstanding?

Reply:
If you're using compiler 10.0 or earlier, this is still required. I'll say that explicitly.